### PR TITLE
refactor(frontend): remove email in wallet

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
Do not use and store email in wallet anymore

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: frontend/src/components/Menu/Navbar.spec.js [hunk 1]
```diff
@@ -64,8 +64,8 @@ describe('AuthNavbar', () => {
         )
       })
 
-      it('has the email address', () => {
-        // expect(wrapper.find('div.small:nth-child(2)').text()).toBe(wrapper.vm.$store.state.email)
+      // I think this should be username
```
Comment (mahula) [inline frontend/src/components/Menu/Navbar.spec.js:67]: Good point.

File: frontend/src/layouts/DashboardLayout.spec.js [hunk 2]
```diff
@@ -260,34 +259,6 @@ describe('DashboardLayout', () => {
         })
       })
 
-      describe.skip('elopage URI', () => {
```
Comment (mahula) [inline frontend/src/layouts/DashboardLayout.spec.js:263]: Finally removing this.

**Review by mahula — APPROVED** [review]
Nice.
I approve.

File: frontend/src/pages/TransactionLink.spec.js [hunk 3]
```diff
@@ -101,7 +101,7 @@ describe('TransactionLink', () => {
               validUntil: transactionLinkValidExpireDate(),
               redeemedAt: '2022-03-18T10:08:43.000Z',
               deletedAt: now,
-              user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+              user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'other-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:104]: isn't there a validation on the gradidoID if it has the correct format and version?

File: frontend/src/pages/TransactionLink.spec.js [hunk 4]
```diff
@@ -132,7 +132,7 @@ describe('TransactionLink', () => {
               validUntil: '2020-03-18T10:08:43.000Z',
               redeemedAt: '2022-03-18T10:08:43.000Z',
               deletedAt: null,
-              user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+              user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'other-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:135]: see above uuid format and version validation?

File: frontend/src/pages/TransactionLink.spec.js [hunk 5]
```diff
@@ -163,7 +163,7 @@ describe('TransactionLink', () => {
               validUntil: transactionLinkValidExpireDate(),
               redeemedAt: '2022-03-18T10:08:43.000Z',
               deletedAt: null,
-              user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+              user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'other-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:166]: see above uuid format and version validation?

File: frontend/src/pages/TransactionLink.spec.js [hunk 6]
```diff
@@ -195,7 +195,7 @@ describe('TransactionLink', () => {
               validUntil: transactionLinkValidExpireDate(),
               redeemedAt: null,
               deletedAt: null,
-              user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+              user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'other-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:198]: uuid format and version?

File: frontend/src/pages/TransactionLink.spec.js [hunk 7]
```diff
@@ -239,7 +239,7 @@ describe('TransactionLink', () => {
                   validUntil: transactionLinkValidExpireDate(),
                   redeemedAt: null,
                   deletedAt: null,
-                  user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+                  user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'current-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:242]: uuid format and version?

File: frontend/src/pages/TransactionLink.spec.js [hunk 8]
```diff
@@ -275,7 +275,7 @@ describe('TransactionLink', () => {
                   validUntil: transactionLinkValidExpireDate(),
                   redeemedAt: null,
                   deletedAt: null,
-                  user: { firstName: 'Peter', publisherId: 0, email: 'peter@listig.de' },
+                  user: { firstName: 'Peter', publisherId: 0, gradidoID: 'other-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:278]: uuid format and version?

File: frontend/src/pages/TransactionLink.spec.js [hunk 9]
```diff
@@ -351,7 +351,7 @@ describe('TransactionLink', () => {
                 validUntil: transactionLinkValidExpireDate(),
                 redeemedAt: null,
                 deletedAt: null,
-                user: { firstName: 'Bibi', publisherId: 0, email: 'bibi@bloxberg.de' },
+                user: { firstName: 'Bibi', publisherId: 0, gradidoID: 'other-user-id' },
```
Comment (clauspeterhuebner) [inline frontend/src/pages/TransactionLink.spec.js:354]: uuid format and version?

**Review by clauspeterhuebner — COMMENTED** [review]
some comments and questions

File: frontend/src/pages/TransactionLink.spec.js (hunk 9 above)
Comment (Mogge) [reply frontend/src/pages/TransactionLink.spec.js:354]: Is validated in the backend.

**Pull request merged**